storybook/storybook/src/packages/help/src/help-custom.scss

```diff
@@ -8,16 +8,6 @@
       background: white;
       box-shadow: 2px 0 5px 0px rgba(0, 0, 0, 0.2);
       padding: 0;
-      &[data-placement^='right'] {
```

Comment:
this was used by the story : https://axaguildev.github.io/react-toolkit/latest/storybook/index.html?path=/story/help--html to set arrow color to white.
![image](https://user-images.githubusercontent.com/43414022/84241924-72a53200-ab00-11ea-89e9-5c01cd1146bc.png)

Now, the arrow is blue :
![image](https://user-images.githubusercontent.com/43414022/84241811-4b4e6500-ab00-11ea-9327-c42a9806c4f3.png)